### DISCUSSION TITLE
Field Type: Business Hours

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#16756] [Feature Requests]
It it possible to create a business hours in the database. Currently my idea is that i create a collection "businesshours". It consits of:
- starttime: Time
- endtime: Time
- dayofweek: RadioButton (String)

I think it would be easier to have a custom field for these type of informations. Any recommendations?

**Comment by NilsBaumgartner1994** [comment]
Also it would be nice to have a DayOfTheWeek Field, which can select from the weekdays and weekends.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
